frontend/src/lib/layout/DetailsPage.svelte

```diff
@@ -16,6 +16,8 @@
     children,
     ...rest
   }: Props = $props();
+
+  const headerContentRender = $derived(headerContent);
```

Comment:
For next time: you probably could have made this change directly in the list of props with: `headerContent: headerContentRender,`. I.e. you can rename a property like that.

Reply:
I'll double check, but I suspect that would have still triggered the infinite update loop, i.e. the key to breaking the loop was not to rename it but to have a different object. But if renaming it was all that was needed, then yes, that would have been simpler.